resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/ClientSuite.scala

```diff
@@ -206,6 +214,75 @@ class ClientSuite extends SparkFunSuite with Matchers {
     appContext.getPriority.getPriority should be (1)
   }
 
+  test("specify a more specific type for the application") {
+    // Limit to 20 characters
```

Comment:
this comment only applies to 4 below. Can you update this to say something like test 20 character limit, yarn should truncate it over 20 characters.